qa/integration/pom.xml

```diff
@@ -233,7 +233,14 @@
             <artifactId>kapua-translator-kura-jms</artifactId>
             <scope>provided</scope>
         </dependency>
-
+        <dependency>
```

Comment:
Please change header to: "    Copyright (c) 2017, 2020 Eurotech and/or its affiliates and others"

Reply:
I changed all the requested headers.